src/Paramore.Brighter/ApplicationTelemetry.cs

```diff
@@ -23,11 +23,13 @@
 #endregion
 
 
+using System;
 using System.Diagnostics;
 using System.Reflection;
 
 namespace Paramore.Brighter
 {
+    [Obsolete("Use BrighterTracer instead")]
```

Comment:
If this is going into master should we just Remove this class (For v10)

Reply:
Will happen in a later PR as part of cleanup